### ISSUE TITLE
Will Object_detection models add yolo/yolo2 ?

### ISSUE THREAD

**Issue by xhzcyc** [tensorflow/models#1880]
The yolo/yolo2 models also have good performance. Will you add them into the object_detection models?

**Comment by jch1** [comment]
Hi @xhzcyc - I agree that they have great performance. At a high level they should be implementable by subclassing the SSDMetaArch class possibly with a few modifications, e.g. with a slightly different loss function, different anchors, and different feature extractor.

However we are a bit limited in terms of bandwidth for doing this and testing it, so there is no concrete plan to add these models unless someone volunteers to do most of the heavy lifting.

**Comment by VastoLorde95** [comment]
@jch1 I will be taking this feature request. You can assign this to me.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
